servicetalk-http-utils/src/main/java/io/servicetalk/http/utils/TimeoutHttpRequesterFilter.java

```diff
@@ -90,20 +71,6 @@ public TimeoutHttpRequesterFilter(final Duration duration, final boolean fullReq
         this(simpleDurationTimeout(duration), fullRequestResponse);
     }
 
-    /**
-     * Creates a new instance.
-     *
-     * @param duration the timeout {@link Duration}
-     * @param fullRequestResponse if true then timeout is for full request/response transaction otherwise only the
-     * response metadata must arrive before the timeout.
-     * @param timeoutExecutor the {@link Executor} to use for managing the timer notifications
-     */
-    public TimeoutHttpRequesterFilter(final Duration duration,
-                                      final boolean fullRequestResponse,
-                                      final Executor timeoutExecutor) {
```

Comment:
This looks like a useful ctor to me when users don't want to care about `TimeoutFromRequest` function.